examples/react-widget-app/src/App.css

```diff
@@ -0,0 +1,54 @@
+html {
+    background-color: #F5F0E9;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
I think we don't need this, since there is a font definition on widget... if we need this for the other divs you created, better to use the same font that we are using on the widget